.github/workflows/delete_stale_branches.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v4
+        uses: actions/checkout@v3
```

Comment:
We shouldn't go backwards a version here

Reply:
hmm - good catch, I was just pulling the k8s assuming it was getting all the latest from master ...